src/tree/EnvironmentTreeItem.ts

```diff
@@ -42,62 +31,57 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
     public actionsTreeItem: ActionsTreeItem;
     public appSettingsTreeItem: AppSettingsTreeItem;
     public functionsTreeItem: FunctionsTreeItem;
-    public readonly data: StaticEnvironment;
+    public readonly data: WebSiteManagementModels.StaticSiteBuildARMResource;
+
+    public name: string;
+    public id: string;
+    public label: string;
+    public repositoryUrl: string;
+    public branch: string;
+    public buildId: string;
+
+    public isProduction: boolean;
```

Comment:
Mostly just to clean it up a little bit because the new data types required me to use `nonNullProp` for everything.

Also once these are set, there probably isn't any reason that they'd change on a refresh, so using the getters seemed a little needless.